.github/workflows/build.yml

```diff
@@ -25,16 +25,12 @@ jobs:
             cachepath: |
               ~/Library/Caches/go-build
               ~/go/pkg/mod
-          - os: windows-latest
-            cachepath: |
-              %LocalAppData%\go-build
-              ~/go/pkg/mod
-    steps:
+    steps:  
 
     - name: Set up Go 1.x
       uses: actions/setup-go@v2
       with:
-        go-version: ^1.15
+        go-version: ~1.15.15
```

Comment:
Any particular reason we have to specify a stricter version of Go here?

Reply:
1.15.15 is the starting point.  `~` says < 1.16.0.  So all patch versions of 1.15.x above starting point.

Reply:
The existing go version was `^1.15`, which unless I'm mistaken, could pull something >=1.16.0

Are we targeting specific releases of Go? Do we need to have compatibility for versions other than 1.15?

Reply:
yes, right now we are only targeting 1.15.x.  `^` was incorrectly being used.